tests/contracts/root_chain/test_process_exits.py

```diff
@@ -274,7 +274,9 @@ def test_finalize_challenged_exit_will_not_send_funds(testlang):
     testlang.forward_timestamp(2 * MIN_EXIT_PERIOD + 1)
 
     pre_balance = testlang.get_balance(testlang.root_chain)
+    testlang.flush_events()
     testlang.process_exits(NULL_ADDRESS, 0, 1, sender=finalizer.key)
+    assert [] == testlang.flush_events()
```

Comment:
I'd like to challenge a bit here: Is the logic of emitting Event really something we'd like to test? (well...I can understand there are cases that clients need to rely on Event to get the data they need then this might be important, not really sure about the case here though?)

However, if we like to test this, might be easier for people to understand if we test that feature on a separate test.
eg. `test_finalize_challenged_exit_will_not_emit_event`. It is easy to read this test in this PR but might be really hard for people to understand why are we asserting on empty event in the future.

Reply:
+1